### ISSUE TITLE
Some shortcuts from the Script Lab Editor are already taken by the host app

### ISSUE THREAD

**Issue by maninweb** [OfficeDev/script-lab#698]
Hi there,
I noticed, that some shortcuts, which are indicated in the context menu of the editor, are already taken by the host app - I tested Excel, Word and PowerPoint. 

![image](https://user-images.githubusercontent.com/13045898/54882822-43a09300-4e5e-11e9-8d75-65d25f10472a.png)

Ctrl+F12, Shift+F12 and Alt+F1 do not run the context menu command, but the commands defined by the app and therefore open the Open/Save File Dialog or add a new chart in Excel.
In PowerPoint, Ctrl/Shift+F12 open the dialogs and Alt+F1 works as expected in the Script Lab editor.
In Word, the 3 shortcuts mentionned before seem interestingly to do nothing. Closing Script Lab or setting the focus to the Word Doc and then using Strg/Shift + F12 then calls the Open/Save dialogs.

I did that test on a Windows 7 German Computer and in Office 2016 firstly set to German and then set to English.

Best,
Mourad

**Comment by nico-bellante** [comment]
Hi Mourad,

Thank you so much for your time in finding and filing all of these bugs. :) 

While we were aware that the host apps are eating some of the shortcut keys, we don't have a great way of resolving that issue at the moment. (Ideally, you'd want the hotkey to apply to whichever had the focus.) Our current workaround has been to offer alternative hotkeys that we've found don't conflict with any of the host apps. 

I think that the behavior you've pictured above can be very misleading as it looks like those hotkeys should trigger those actions. (Some of which I don't think would work for a lot of things anyways in our case such as go to definition for something defined in a library.) 

Either the shortcuts should be removed as to not be misleading, or we should change them to something that will actually work.

**Comment by Zlatkovsky** [comment]
Adding @peronald .  This seems essentially a dup of https://github.com/OfficeDev/script-lab/issues/331.  I think we need to re-bubble-this up to the platform, esp. as more scenarios involving Monaco Editor encounter this...

**Comment by wandyezj** [comment]
This appears to be fixed.
